vue.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     markdownRule.test(/\.md/).use("raw-loader").loader("raw-loader");
   },
   devServer: {
-    // proxy: "https://nest.land",
+    proxy: "https://nest.land",
```

Comment:
Do we need this?

Reply:
Doesn't matter if it is there, it is not uses in production